tsl/src/continuous_aggs/create.c

```diff
@@ -1562,10 +1597,12 @@ fixup_userview_query_tlist(Query *userquery, List *tlist_aliases)
  *
  * Notes: ViewStmt->query is the raw parse tree
  * panquery is the output of running parse_anlayze( ViewStmt->query)
+ * multiple_cagg = true => we already have a cagg on the raw hypertable.
+ *               so don't recreate invalidation trigger.
  */
 static void
 cagg_create(ViewStmt *stmt, Query *panquery, CAggTimebucketInfo *origquery_ht,
-			WithClauseResult *with_clause_options)
+			WithClauseResult *with_clause_options, bool multiple_cagg)
```

Comment:
the `multiple_cagg` parameter does not seem to be used in this function

Reply:
removed.